src/software/ai/hl/stp/tactic/kickoff_chip_tactic.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include "software/ai/hl/stp/tactic/tactic.h"
+
+/**
+ * The KickoffChipTactic will move the assigned robot to the given chip origin and then
+ * chip the ball to the chip target.
+ */
```

Comment:
Quick comment above what differentiates this from the `ChipTactic` would be super helpful here, so that someone in the future doesn't try to revert our change :stuck_out_tongue:

Reply:
Done